### DISCUSSION TITLE
How to disable font ligatures?

### DISCUSSION THREAD

**Discussion by felipecrs** [microsoft/terminal#15270] [Q&A] [answered]
I'm a little confused. I have:

```json
"font": {
  "face": "FiraCode Nerd Font Mono",
  "features": {
    "liga": 0
  }
}
```

But ligatures doesn't go away. Am I missing something?

- https://learn.microsoft.com/en-us/windows/terminal/customize-settings/profile-appearance#font-features

**Comment by zadjii-msft** [comment]
From https://github.com/microsoft/terminal/issues/13116#issuecomment-1128706740

> I was just testing this out locally. Apparently, to disable ligatures, you need
> 
> ```js
> "font": {
>     "face": "Fira Code",
>     "features": {
>         "liga": 0,
>         "calt": 0
>     }
> },
> ```
> 
> Both `"liga": 0` and `"calt": 0` are apparently needed for Fira Code. Does that work for you/?

**Reply by felipecrs** [reply]
That works, thanks!

**Comment by n0099** [comment, marked as answer]
From https://github.com/microsoft/terminal/issues/13116#issuecomment-2148133660
> Good news! In Terminal 1.21 (which is currently in [Preview](https://aka.ms/terminal-preview)), you can now configure these things through the settings UI.
> 
> ![Image](https://github.com/microsoft/terminal/assets/189190/fc15e535-cc1d-48a3-9235-8d9a398a3bb9)
